tests/test_product_pages_api.py

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: GPL-3.0+
+import requests_mock
+
+from retasc.product_pages_api import ProductPagesApi
+
+PP_URL = "https://product_pages.example.com"
+PP_API = ProductPagesApi(PP_URL)
```

Comment:
This will not work well due to the caching. It would be better to create the object for each test method, e.g. using a pytest fixture:

```python
from pytest import fixture
...

@fixture
def pp_api():
    return ProductPagesApi(PP_URL)

...

def test_get_pp_url_info(pp_api):
    assert pp_api.api_url == PP_URL
```